### PR TITLE
Fix ingress documentation

### PR DESCRIPTION
**Pull request by rtribotte**
### What does this PR do?

This PR adds a TLS example for the Ingress provider, clarifies the usage of the TLS section in the Ingress Spec, cleans unnecessary elements in the first example, and re-orders elements in the TLS documentation section. 

### Motivation

Clarify how to configure an Ingress.
related to #7420

### More

- [ ] ~Added/updated tests~
- [x] Added/updated documentation

### CODE REVIEW

File: docs/content/routing/providers/kubernetes-ingress.md [hunk 1]
```diff
@@ -382,7 +528,9 @@ and will connect via TLS automatically.
             backend:
               serviceName: service1
               servicePort: 80
-    
+      # Only loading certificates from Secrets, that will be used to terminate TLS traffic.
```
Comment (SantoDE) [inline docs/content/routing/providers/kubernetes-ingress.md:531]:
```suggestion
Only selects which certificate(s) should be loaded from the secret, in order to terminate TLS
```

**Review by SantoDE — CHANGES_REQUESTED** [review]
Only 1 minor nitpick 👼

**Review by jbdoumenjou — APPROVED** [review]
LGTM

File: docs/content/routing/providers/kubernetes-ingress.md (hunk 1 above)
Comment (rtribotte) [reply docs/content/routing/providers/kubernetes-ingress.md]: @SantoDE I made the changes :-)

File: docs/content/routing/providers/kubernetes-ingress.md [hunk 2]
```diff
@@ -340,27 +331,379 @@ Please see [this documentation](https://kubernetes.io/docs/concepts/services-net
...
-Traefik automatically requests endpoint information based on the service provided in the ingress spec.
-Although Traefik will connect directly to the endpoints (pods),
-it still checks the service port to see if TLS communication is required.
+TLS can be enabled through the [HTTP options](../entrypoints.md#tls) of an Entrypoint:
 
-There are 3 ways to configure Traefik to use https to communicate with pods:
+```bash tab="CLI"
+# Static configuration
+-- entrypoints.websecure.address=:443
+-- entrypoints.websecure.http.tls
```
Comment (ldez) [inline docs/content/routing/providers/kubernetes-ingress.md:341]:
```suggestion
--entrypoints.websecure.address=:443
--entrypoints.websecure.http.tls
```

**Review by ldez — CHANGES_REQUESTED** [review]

**Review by SantoDE — APPROVED** [review]
LGTM

**Review by ldez — APPROVED** [review]
LGTM

**Pull request merged**